homeassistant/components/backup/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""The Backup integration."""
+from homeassistant.components.hassio import is_hassio
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN, LOGGER
+from .http import async_register_http_views
+from .manager import BackupManager
+from .websocket import async_register_websocket_handlers
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Backup integration."""
+    if is_hassio(hass):
+        LOGGER.warning(
+            "The backup integration is not supported on this installation method, "
+            "please remove it from your configuration"
+        )
+        return False
+
+    hass.data[DOMAIN] = manager = BackupManager(hass)
+
+    if not manager.backup_dir.exists():
+        LOGGER.debug("Creating backup directory")
+        hass.async_add_executor_job(manager.backup_dir.mkdir)
+
+    await manager.load_backups()
```

Comment:
We shouldn't load backups during startup. We should load it the first time the page is visited / the user interacts with the backup manager.

Reply:
![image](https://user-images.githubusercontent.com/1444314/153738710-176125d7-129f-456a-94d5-9d0aad72e00d.png)